packages/CopyInput/src/CopyInputInputPropsCollector.js

```diff
@@ -0,0 +1,6 @@
+// Shell component for enhancing DX (developer experience)
+export default function CopyInputInputPropsCollector() {
```

Comment:
`nit` I would rename this one to Input there is not really a reason to describe it as a PropCollector. Just a few other components are named like that, for example, the DatePicker was renamed by Allison a couple of days ago. From PropCollectors to just Input, cc @mikrotron